fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/drawer/Drawer.kt

```diff
@@ -353,7 +355,6 @@ private fun HorizontalDrawer(
     drawerShape: Shape,
     drawerElevation: Dp,
     drawerBackground: Brush,
-    drawerContentColor: Color,
```

Comment:
Not used anymore?

Reply:
Surface had a param for this but Box doesn't and it was hardcoded to be transparent. Removing as there is no impact from this.